### PR TITLE
🐛 Fixes PKI/SSL check for minikube

### PR DESCRIPTION
**Pull request by czunker**
Signed-off-by: Christian Zunker <christian@mondoo.com>

### CODE REVIEW

**Comment by czunker** [comment]
I have read the Mondoo CLA Document and I hereby sign the CLA

File: core/mondoo-kubernetes-security.mql.yaml [hunk 1]
```diff
@@ -618,6 +618,11 @@ queries:
         user.name == "root"
         group.name == "root"
       }
+    } else if (file("/var/lib/minikube/certs/").exists) {
```
Comment (joelddiaz) [inline core/mondoo-kubernetes-security.mql.yaml:621]: It seems entirely possible to spread all the various certificate files around the filesystem, and just point the kube-apiserver to the appropriate places with the corresponding params. But if we're going to try to make an assertion on at least one of the parent directories holding these files:

Can we just extract the parent directory out of the kube-apiserver processes params (something like `--client-ca-file`) to make this check more generic?

Comment (tas50) [reply core/mondoo-kubernetes-security.mql.yaml:621]: @joelddiaz would that be an api server equiv of the kubelet resource?

Comment (joelddiaz) [reply core/mondoo-kubernetes-security.mql.yaml:621]: AFAICT, this is more of a kube-apiserver kind of thing. Looking at an invocation of kube-apiserver, you can see

`kube-apiserver --advertise-address=192.168.39.109 --allow-privileged=true --authorization-mode=Node,RBAC --client-ca-file=/var/lib/minikube/certs/ca.crt --enable-admission-plugins=NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota --enable-bootstrap-token-auth=true --etcd-cafile=/var/lib/minikube/certs/etcd/ca.crt --etcd-certfile=/var/lib/minikube/certs/apiserver-etcd-client.crt --etcd-keyfile=/var/lib/minikube/certs/apiserver-etcd-client.key --etcd-servers=https://127.0.0.1:2379 --kubelet-client-certificate=/var/lib/minikube/certs/apiserver-kubelet-client.crt --kubelet-client-key=/var/lib/minikube/certs/apiserver-kubelet-client.key --kubelet-preferred-address-types=InternalIP,ExternalIP,Hostname --proxy-client-cert-file=/var/lib/minikube/certs/front-proxy-client.crt --proxy-client-key-file=/var/lib/minikube/certs/front-proxy-client.key --requestheader-allowed-names=front-proxy-client --requestheader-client-ca-file=/var/lib/minikube/certs/front-proxy-ca.crt --requestheader-extra-headers-prefix=X-Remote-Extra- --requestheader-group-headers=X-Remote-Group --requestheader-username-headers=X-Remote-User --secure-port=8443 --service-account-issuer=https://kubernetes.default.svc.cluster.local --service-account-key-file=/var/lib/minikube/certs/sa.pub --service-account-signing-key-file=/var/lib/minikube/certs/sa.key --service-cluster-ip-range=10.96.0.0/12 --tls-cert-file=/var/lib/minikube/certs/apiserver.crt --tls-private-key-file=/var/lib/minikube/certs/apiserver.key`

The `/var/lib/minikube/certs` directory shows up multiple times for multiple different CLI parameters.

Whereas when I browse the kubelet config, I can only find one reference to Authentication->x509->clientCAFile pointing to anything inside of `var/lib/minikube/certs`.

**Review by joelddiaz — APPROVED** [review]

**Pull request merged**